test/cli/validate-local-bad-element3/command

```diff
@@ -0,0 +1 @@
+validate 
```

Comment:
Maybe I haven't gotten to where these are used yet, but what is the point of all these files with just the string `validate` in them?